applications/rubintv/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for rubintv.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- A special tag for letting the scripts know where they are
+# running.  Must be overridden at each site
+siteTag: ""
+
+frontend:
+  # -- If set to true, enable more verbose logging.
+  debug: false
+  # -- Prefix for rubintv's frontend API routes.
+  pathPrefix: "/rubintv"
+  # -- Settings for rubintv OCI image
+  image:
+    # -- rubintv frontend image to use
+    repository: "ghcr.io/lsst-sqre/rubintv"
+    # -- Pull policy for the rubintv image
+    pullPolicy: "IfNotPresent"
+    # -- Tag of rubintv image to use
+    # @default -- The appVersion of the chart
+    tag: ""
+  # -- Resource limits and requests for the rubintv frontend pod
+  resources: {}
+  # -- Annotations for the rubintv frontend pod
+  podAnnotations: {}
+  # -- Node selector rules for the rubintv frontend pod
+  nodeSelector: {}
+  # -- Tolerations for the rubintv frontend pod
+  tolerations: []
+  # -- Affinity rules for the rubintv frontend pod
+  affinity: {}
+
+workers:
+  # -- how many replicas to use
+  replicas: 5
+  # -- If set to true, enable more verbose logging.
+  debug: false
+  # -- Prefix for the (internal) worker API routes
+  pathPrefix: "/"
+  # -- UID to run as (site-dependent, because of filesystem perms)
+  uid: 1000
+  # -- GID to run as
+  gid: 1000
+  # We will try to avoid fsid; just use uid.  Might need to change that.
+  # -- Settings for OCI image for worker pods
+  image:
+    # -- The Docker registry name for the container image.
+    repository: ts-dockerhub.lsst.org/rubintv-broadcaster
+    # -- The tag of the container image to use.
+    tag: develop
+    # -- The policy to apply when pulling an image for deployment.
+    pullPolicy: IfNotPresent
+  # -- Script that runs in RUN_ARG.  This needs to be replaced with
+  # something less scary
+  script: slac/rubintv/workerPod1.py
+  # -- This section holds a list of key, value pairs for environmental
+  # variables (name: key, value: value).
+  # NOTE: RUN_ARG is taken care of by the chart using _script_.
+  env: []
+  # -- This section holds specifications for secret injection.
+  # If this section is used, each object listed must have the following attributes defined:
+  # _name_ (The label for the secret),
+  # _secretName_ (The name of the vault store reference. Uses the _namespace_ attribute to construct the full name),
+  # _secretKey_ (The key in the vault store containing the necessary secret)
```

Comment:
Here and below, these complex comments are not going to render in any useful way. I think it would be better to put an example into a comment, and then have the documentation comment say to look at `values.yaml` for an example.